app/models/planet.py

```diff
@@ -0,0 +1,35 @@
+from app import db
+
+'''
+Defined a Planet model with the attributes id, title,
+and description, and moons. Created instance method, update, to update our model. 
+Class method, create, to create a new instance of planet.
+'''
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    description = db.Column(db.String)
+    moons = db.Column(db.Boolean, nullable=False)
+
+    def to_json(self):
+        return {
+            "id": self.id,
+            "title": self.title,
+            "description": self.description,
+            "moons": self.moons
+        }
+
+    def update(self, request_body):
+        self.title = request_body["title"]
+        self.description = request_body["description"]
+        self.moons = request_body["moons"]
+
+    @classmethod
+    def create(cls,request_body):
+        new_planet = cls(
+            title=request_body["title"],
+            description=request_body["description"],
+            moons = request_body["moons"]
+        )
+        return new_planet
```

Comment:
Nice work pulling these two methods into the model Planet to keep your routes short and sweet. 

This comment applies to your create() and update() methods:

How would you add in validation to make sure that all the required fields are sent in the request to your server?

For example, if someone sent a POST request but left off moons, then line 33 would throw KeyError that it couldn't find moons.

it would be nice to handle the error and return a message so that the client knows their request was invalid and they need to include moons. Something to think about for Task List.